servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/StreamObserverTest.java

```diff
@@ -124,6 +124,7 @@ void teardown() throws Exception {
         safeClose(client);
     }
 
+    @SuppressWarnings("PMD.AvoidPrintStackTrace")
```

Comment:
Any reason not to use a logger below?

Reply:
This method was reused in many different classes, it was awkward to use a logger from `StreamObserverTest` for all those cases. I moved both "safe*" methods in a new test fixture: https://github.com/apple/servicetalk/pull/2753/commits/3667b92e766b4b3563aa8bc203ab6aaae82e59ff